blockchain/subsidy_test.go

```diff
@@ -51,7 +51,7 @@ func TestBlockSubsidy(t *testing.T) {
 		}
 	}
 
-	expectedSubsidy := int64(2043784000590848) + mainnet.BlockOneSubsidy()
+	expectedSubsidy := int64(2039990080590848) + mainnet.BlockOneSubsidy()
```

Comment:
This change is caused by change in initial Ticket Price and Stake Validation Height. The first one increase Total Subsidy in time but the second one decrease it.